includes/admin.inc

```diff
@@ -5,29 +5,222 @@
-    '#description' => t('In order to designate the domain that will host your 
-      articles you must add your Facebook page ID to a metatag in the HEAD tag 
-      of your HTML page. See <a href="@claim_url">Claiming your URL</a>.', $args),
+    '#description' => t('In order to designate the domain that will host your
+      articles you must add your Facebook page ID to a metatag in the HEAD tag
+      of your HTML page. Entering your Facebook Page ID here will add the
+      metatag automatically. See <a href="@claim_url">Claiming your URL
+      </a>.', $args),
   );
 
+  // Add the style configuration.
+  $form['fb_instant_articles_style'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Article Style'),
+    '#default_value' => variable_get('fb_instant_articles_style', 'default'),
+    '#size' => 30,
+    '#element_validate' => array('fb_instant_articles_validate_style'),
+    '#description' => t('Assign your Instant Articles a custom style. To begin, customize a template using the <a href="@style_url" target="_blank">Style Editor</a>. Next, input the name of the style below. <strong>Note</strong>: if this field is left blank, the module will enable the “Default” style. Learn more about Instant Articles style options in the <a href="@design_url" target="_blank">Design Guide</a>.', array('@style_url' => '', '@design_url' => 'https://developers.facebook.com/docs/instant-articles/guides/design')),
+  );
+
+  // Add the Ads sub-section.
+  $form = fb_instant_articles_module_config_ads($form);
+
+  // Add the Analytics sub-section.
+  $form = fb_instant_articles_module_config_analytics($form);
+
+  // Add the Debug Configuration.
   $form['fb_instant_articles_enable_logging'] = array(
     '#type' => 'checkbox',
     '#title' => t('Enable FB Instant Articles SDK logging?'),
     '#default_value' => variable_get('fb_instant_articles_enable_logging'),
     '#description' => t('Sends Facebook Instant Articles SDK logging messages to Drupal watchdog.'),
   );
 
-  return system_settings_form($form);
+  $form = system_settings_form($form);
+  return $form;
+}
+
+/**
+ * Generates Ads sub-section of Module Configuration section
+ */
+function fb_instant_articles_module_config_ads($form) {
+  $form['ads'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Ads'),
+    '#collapsible' => TRUE,
+    '#description' => t('Choose your preferred method for displaying ads in your Instant Articles and input the code in the boxes below. Learn more about your options for <a href="@ads_url">advertising in Instant Articles</a>', array('@ads_url' => 'https://developers.facebook.com/docs/instant-articles/ads-analytics')),
+  );
+
+  $form['ads']['fb_instant_articles_ad_type'] = array(
+    '#type' => 'select',
+    '#title' => t('Ad Type'),
+    '#default_value' => variable_get('fb_instant_articles_ad_type', FB_INSTANT_ARTICLES_AD_TYPE_NONE),
+    '#options' => array(
+      FB_INSTANT_ARTICLES_AD_TYPE_NONE => FB_INSTANT_ARTICLES_AD_TYPE_NONE,
```

Comment:
Thanks for that input. If you see my commit message [over here](https://github.com/BurdaMagazinOrg/module-fb_instant_articles/pull/36/commits/b52ca3350f8076fe0dab4d49bad3f9ca09902803), I actually made a note that we should follow up an change this behavior a bit (so we don't store the constant values in db, but the constant names (or something more static)). I'll file a d.o issue on this.